packages/devtools_app/lib/src/screens/vm_developer/object_inspector/vm_object_model.dart

```diff
@@ -265,6 +265,46 @@ class CodeObject extends VmObject {
 
   @override
   String? get name => obj.name;
+
+  /// A collection of CPU profiler information for individual [Instruction]s.
+  ///
+  /// Returns null if the CPU profiler is disabled.
+  CpuProfilerTicksTable? get ticksTable => _table;
+  CpuProfilerTicksTable? _table;
+
+  @override
+  Future<void> initialize() async {
+    await super.initialize();
+
+    final service = serviceManager.service!;
+    final isolateId = serviceManager.isolateManager.selectedIsolate.value!.id!;
+
+    // Attempt to retrieve the CPU profile data for this code object.
+    try {
+      final samples = await service.getCpuSamples(isolateId, 0, ~0);
+      final codes = samples.codes;
+
+      ProfileCode? match;
```

Comment:
this can be
`ProfileCode? match = codes.firstWhereOrNull((profileCode) => profileCode.code == ref);`